hooks/post-command

```diff
@@ -69,6 +70,12 @@ do
     ENTRIES[$COUNTER,2]="DetailType=${!ENTRY_DETAIL_TYPE}"
   fi
 
+  if [ -n "${!ENTRY_EVENT_BUS_NAME_ENV}" ]
+  then
+    ENTRY_EVENT_BUS_NAME_ENV_NAME=${!ENTRY_EVENT_BUS_NAME_ENV}
+    export ${NUMBERED_PREFIX}EVENT_BUS_NAME="${!ENTRY_EVENT_BUS_NAME_ENV_NAME}"
```

Comment:
could you do the export directly though? 
`export ${NUMBERED_PREFIX}EVENT_BUS_NAME="${ENTRY_EVENT_BUS_NAME_ENV}"`

Reply:
add test case for clarity